scripts/2.TCGA-MLexample-covariates.py

```diff
@@ -76,146 +75,166 @@
 
 # In[9]:
 
-# Here are the percentage of tumors with NF1
+# Here are the percentage of tumors with TP53
 y.value_counts(True)
 
 
-# ## Pre-process data set
-# TODO: currently running PCA on both train and test partitions
+# ## Set aside 10% of the data for testing
 
 # In[10]:
 
-# Pre-process expression data for use later
-n_components = 100
-scaled_expression = StandardScaler().fit_transform(expression)
-pca = PCA(n_components).fit(scaled_expression)
-explained_variance = pca.explained_variance_
-expression_pca = pca.transform(scaled_expression)
-expression_pca = pd.DataFrame(expression_pca)
-expression_pca = expression_pca.set_index(expression.index.values)
+X = pd.concat([covariates, expression], axis=1)
+print('Gene expression matrix shape: {0[0]}, {0[1]}'.format(expression.shape))
+print('Covariates matrix shape: {0[0]}, {0[1]}'.format(covariates.shape))
 
+# Typically, this can only be done where the number of mutations is large enough
+X_train, X_test, y_train, y_test = train_test_split(X, y, test_size=0.1, random_state=0)
```

Comment:
IIRC the comment may be along the lines of, when n_positives is small, `test_size` may have to be larger so there are a meaningful number of testing positives. Perhaps remove the comment?

If we're not already, printing out an n_positives, n_negatives versus train, test table here would be good.